types/node/crypto.d.ts

```diff
@@ -122,13 +122,15 @@ declare module "crypto" {
         private constructor();
         update(data: BinaryLike): Hash;
         update(data: string, input_encoding: Utf8AsciiLatin1Encoding): Hash;
+        update(data: Buffer, input_encoding: Utf8AsciiBinaryEncoding): Hash;
```

Comment:
https://nodejs.org/dist/latest-v12.x/docs/api/crypto.html#crypto_hash_update_data_inputencoding

`If data is a Buffer, TypedArray, or DataView, then inputEncoding is ignored.`

Reply:
Thank you.  I had somehow missed the fact the encoding is ignored when a buffer is specified.   Please ignore my changes.